Source/WebKit/GPUProcess/graphics/ScopedWebGLRenderingResourcesRequest.cpp

```diff
@@ -52,14 +56,16 @@ void ScopedWebGLRenderingResourcesRequest::scheduleFreeWebGLRenderingResources()
 
 void ScopedWebGLRenderingResourcesRequest::freeWebGLRenderingResources()
 {
+#if USE(ANGLE)
```

Comment:
This should be scoped with ENABLE(WEBGL) as there is no WebGL in GPUP without ANGLE as per #4374